### ISSUE TITLE
Problem installing 1.2.1 with build.sh

### ISSUE THREAD

**Issue by pedroroberto** [Waikato/wekaDeeplearning4j#14]
Running "./build.sh -v -b GPU -c -i" on wekaDeeplearning4j-1.2.1 gives me following error. 

```
rm: /usr/local/Caskroom/weka/3.9.1/weka-3-9-1/packages/wekaDeeplearning4jCPU-dev: No such file or directory
rm: /usr/local/Caskroom/weka/3.9.1/weka-3-9-1/packages/wekaDeeplearning4jGPU-dev: No such file or directory
java.io.FileNotFoundException: dist/wekaDeeplearning4jGPU-1.2.0-dev-macosx-x86_64.zip (No such file or directory)
	java.util.zip.ZipFile.open(Native Method)
	java.util.zip.ZipFile.<init>(ZipFile.java:219)
	java.util.zip.ZipFile.<init>(ZipFile.java:149)
	java.util.zip.ZipFile.<init>(ZipFile.java:163)
	weka.core.packageManagement.DefaultPackageManager.getPackageArchiveInfo(DefaultPackageManager.java:354)
	weka.core.WekaPackageManager.installPackageFromArchive(WekaPackageManager.java:2136)
	weka.core.WekaPackageManager.main(WekaPackageManager.java:2839)

	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:219)
	at java.util.zip.ZipFile.<init>(ZipFile.java:149)
	at java.util.zip.ZipFile.<init>(ZipFile.java:163)
	at weka.core.packageManagement.DefaultPackageManager.getPackageArchiveInfo(DefaultPackageManager.java:354)
	at weka.core.WekaPackageManager.installPackageFromArchive(WekaPackageManager.java:2136)
	at weka.core.WekaPackageManager.main(WekaPackageManager.java:2839)
```

It look likes it tries to extract the zip with the wrong name (1.2.0 instead of 1.2.1)

**Comment by slang03** [comment]
Forgot to bump the version in the build file. Fixed with 654c8bcb9d0f554c4474b70e7b67faa7990df579

**Comment by pedroroberto** [comment]
Perferct, good job.
